.gitignore

```diff
@@ -1,2 +1,5 @@
 tags
 .dir-locals.el
+.project
+.cproject
+*.swp
```

Comment:
these are unrelated to this issue...

move this edit to a different PR against #858